generators/create-react-app/templates/circleci.yaml.ejs

```diff
@@ -115,14 +115,14 @@ jobs:
           tag: << pipeline.git.branch >>
           path: <%= packagePath %>
           docker-context: <%= packagePath %>
-          extra_build_args: '--build-arg VERSION=$(git log --max-count 1 --pretty=format:%H "<%= packagePath %>")'
+          extra_build_args: '--build-arg DISABLE_ESLINT_PLUGIN=true --build-arg VERSION=$(git log --max-count 1 --pretty=format:%H "<%= packagePath %>")'
       - docker/build:
           registry: $DOCKER_REGISTRY
           image: <%= projectName %>-<%= packageName %>
           tag: << pipeline.git.branch >>-sentry
           path: <%= packagePath %>
           docker-context: <%= packagePath %>
-          extra_build_args: '--build-arg VERSION=$(git log --max-count 1 --pretty=format:%H "<%= packagePath %>") --target sentry'
+          extra_build_args: '--build-arg DISABLE_ESLINT_PLUGIN=true --build-arg VERSION=$(git log --max-count 1 --pretty=format:%H "<%= packagePath %>") --target sentry'
```

Comment:
This should be as an env variable in the `Dockerfile`, I don't think the will be picked correctly if we passe them only as build args.